app/jobs/bulkrax/child_relationships_job_decorator.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
```

Comment:
@alishaevn is this supposed to be in here?

Reply:
I probably copied it over from another decorator so it may be able to be removed. but it hasn't caused me any issues that I can recall at the moment.